package.json

```
@@ -21,7 +21,9 @@
     "typecheck:vue-tsc": "vue-tsc --noEmit",
     "markdownlint": "markdownlint *.md",
     "electron:build": "cross-env VITE_IS_ELECTRON=true vite build && electron-builder --config electron-builder.config.js",
+    "electron:pack": "cross-env VITE_IS_ELECTRON=true vite build && electron-builder --config electron-builder.config.js --dir",
     "electron:build_pnever": "cross-env VITE_IS_ELECTRON=true vite build && electron-builder --config electron-builder.config.js --publish never",
+    "electron:build_packed": "cross-var electron-builder --config electron-builder.config.js --prepackaged $PREPACKAGED --publish never",
```

別の箇所と命名を合わせると、こんな感じ･･･？（electronに合わせる感じです）
```suggestion
```
```suggestion
    "electron:build": "cross-env VITE_IS_ELECTRON=true vite build && electron-builder --config electron-builder.config.js",
    "electron:build_dir": "cross-env VITE_IS_ELECTRON=true vite build && electron-builder --config electron-builder.config.js --dir",
    "electron:build_pnever": "cross-env VITE_IS_ELECTRON=true vite build && electron-builder --config electron-builder.config.js --publish never",
    "electron:build_pnever_prepack": "cross-var electron-builder --config electron-builder.config.js --publish never --prepackaged $PREPACKAGED",
```

---

https://www.electron.build/#quick-setup-guide
packは一応公式がpackって言ってるんですよね。
electronに合わせても良さそうではある。

---

なるほどです、たしかにpackと書いてますね！

まあ似たようなことするのをbuildとしているので、合わせておくのがまあ丸いかなぁという、結構ゆるめのできればくらいの気持ちです。

---

ここ、どうでしょう。 👀

---

build_pneverとかに合わせるようにしました。試しにビルドしてます： https://github.com/sevenc-nanashi/voicevox/actions/runs/4224768360

---

おっと、windowsだけ落ちてそう･･･？

---

修正したやつをビルド中です： https://github.com/sevenc-nanashi/voicevox/actions/runs/4227464199

---

あの後もう一回修正してビルドしました。
通りました： https://github.com/sevenc-nanashi/voicevox/releases/tag/0.15.0-preview.vite.2

---

おお！！！　あざます、とりあえずここはresolveかなと思いました！